app/src/main/java/com/mobileproto/hireddit/hireddit/GetComment.java

```diff
@@ -0,0 +1,90 @@
+package com.mobileproto.hireddit.hireddit;
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.android.volley.DefaultRetryPolicy;
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.TimeoutError;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.JsonObjectRequest;
+import com.android.volley.toolbox.Volley;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Created by lwilcox on 11/4/2015.
+ */
+
+public class GetComment {
+    public RequestQueue queue;
+    public GetComment(Context context) {
+        queue = Volley.newRequestQueue(context);
+    }
+    public void commentSearch(String searchQuery, final Context context, final CommentCallback callback) {
+
+        String query = searchQuery.replaceAll(" ", "+");
+        Uri.Builder builder = new Uri.Builder();
```

Comment:
I just tested this on my branch and the url was invalid. I'll work on fixing it.